lib/rules/a11y-no-visually-hidden-interactive-element.js

```diff
@@ -0,0 +1,90 @@
+const {getProp, getPropValue} = require('jsx-ast-utils')
+const {getElementType} = require('../utils/get-element-type')
+const {generateObjSchema} = require('eslint-plugin-jsx-a11y/lib/util/schemas')
+
+const defaultClassName = 'sr-only'
```

Comment:
considering making the default `visually-hidden` for readability.

Reply:
I prefer `sr-only` just because it's more widely used within different GitHub projects! As long as we document what the default is and how consumers can change it, I think `sr-only` would be okay.